### PR TITLE
Log HBase version upon Master startup.

### PR DESCRIPTION
**Pull request by anwar6953**
We log the version of several other components, so also logging hbase version.

http://builds.cask.co/browse/CDAP-DUT3867-1

### CODE REVIEW

**Comment by anwar6953** [comment]
This is especially useful when debugging an issue, or when we're dealing with a new or unsupported version of HBase.

**Comment by gokulavasan** [comment]
LGTM :+1:

**Comment by poornachandra** [comment]
Can you add some more versions as mentioned in this JIRA? https://issues.cask.co/browse/CDAP-5218

File: cdap-master/src/main/java/co/cask/cdap/data/runtime/main/ClientVersions.java [hunk 1]
```diff
@@ -24,14 +27,33 @@
 import java.util.regex.Pattern;
 
 /**
- * Provides client versions of infrasturcture components.
+ * Provides client versions of infrastructure components.
  */
 public class ClientVersions {
 
+  public static String getCDAPVersion() {
```
Comment (poornachandra) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/ClientVersions.java]: It will be more readable if this is called `getCdapVersion()`

**Comment by poornachandra** [comment]
LGTM

**Comment by rohitsinha54** [comment]
Flaky failure. New Build running: http://builds.cask.co/browse/CDAP-DUT3867-2

**Pull request merged**